PublicGitArchive/pga-create/Dockerfile

```diff
@@ -0,0 +1,58 @@
+#==========================================================================
+# Stage 1: build pga-create and create select-repos and index-repos scripts
+#==========================================================================
+FROM golang:1.12.0-alpine3.9 AS builder
+
+# build pga-create
+ENV PGA_CREATE_REPO=github.com/src-d/datasets/PublicGitArchive/pga-create
+ENV PGA_CREATE_PATH=$GOPATH/src/$PGA_CREATE_REPO
+COPY . ${PGA_CREATE_PATH}
+RUN go build -tags norwfs -o /bin/pga-create ${PGA_CREATE_PATH}/cmd/pga-create
+
+# select-repos command
+RUN echo -e '#!/bin/sh \n\
```

Comment:
Maybe we can extract these two scripts to files in the repository to make them easier to read ad modify.

Also, instead of using `&&` to continue only when the previous command is successful you can enable exiting the shell on error with:

```
set -e
```

https://linux.die.net/man/1/ash